wasmcloud_host/lib/wasmcloud_host_web/live/start_actor_component.ex

```diff
@@ -0,0 +1,84 @@
+defmodule StartActorComponent do
+  use Phoenix.LiveComponent
+
+  def mount(socket) do
+    {:ok,
+     socket
+     |> assign(:uploads, %{})
+     |> allow_upload(:actor, accept: ~w(.wasm), max_entries: 1)}
+  end
+
+  def handle_event("validate", params, socket) do
+    {:noreply, socket}
+  end
+
+  def handle_event(
+        "start_actor_file",
+        params,
+        socket
+      ) do
+    actor =
+      Phoenix.LiveView.consume_uploaded_entries(socket, :actor, fn %{path: path}, _entry ->
+        replicas = 1..String.to_integer(params["replicas"])
+        {:ok, bytes} = File.read(path)
+        replicas |> Enum.each(fn _ -> HostCore.Actors.ActorSupervisor.start_actor(bytes) end)
+      end)
+
+    {:noreply, socket}
+  end
+
+  def handle_event(
+        "start_actor_ociref",
+        %{"_csrf_token" => _csrf_token, "replicas" => replicas, "actor_ociref" => actor_ociref},
```

Comment:
You should be able to simply leave off the `"_csrf_token"` part of the match. The match should still pass while you're ignoring the CSRF token.

Reply:
Nice, I'll give that a shot